crates/ruff_linter/resources/test/fixtures/ruff/suppressions.py

```diff
@@ -86,3 +86,12 @@ def f():
     # Multiple codes but none are used
     # ruff: disable[E741, F401, F841]
     print("hello")
+
+
+def f():
+    # Unknown rule codes
+    # ruff: disable[YF829]
+    # ruff: disable[F841, RQW320]
```

Comment:
Can you add a test that verifies we don't flag rules defined using [`external`](https://docs.astral.sh/ruff/settings/#lint_external) (might have to be a CLI test)

Reply:
I guess that makes sense